app.rb

```diff
@@ -0,0 +1,151 @@
+require './item'
+require './classes/music_album'
+require './classes/game'
+require './classes/genre'
+
+require './classes/book'
+require './classes/label'
+require './modules/music_album_module'
+require './modules/genres_module'
+require './modules/book_module'
+require './modules/label_module'
+require './modules/games_module'
+require './modules/authors_module'
+
```

Comment:
- [ ] We usually use `require_relative` in case we are importing files within the project folder and `require` when we are importing any `gem` from ruby. So, please in this and all the files Please use `require_relative` wherever you are importing files within the same project folder.

Reply:
Thank you we followed your instructions and replaced `require` to `require_relative` where needed